contents/navigation/navbars/navbar-with-search/index.tsx

```diff
@@ -0,0 +1,190 @@
+import type { FC } from "react"
+import {
+  ChevronRightIcon,
+  LightbulbIcon,
+  PlusIcon,
+  SearchIcon,
+  SquareCheckBigIcon,
+  UserIcon,
+} from "@yamada-ui/lucide"
+import {
+  Avatar,
+  Box,
+  Divider,
+  HStack,
+  IconButton,
+  Indicator,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Kbd,
+  LinkBox,
+  LinkOverlay,
+  Spacer,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+
+const NavbarWithSearch: FC = () => {
+  const tabs = {
+    collections: [
+      { emoji: "👍", label: "Sales" },
+      { emoji: "🚚", label: "Deliveries" },
+      { emoji: "💸", label: "Discounts" },
+      { emoji: "💰", label: "Profits" },
+      { emoji: "✨", label: "Reports" },
+      { emoji: "🛒", label: "Orders" },
+      { emoji: "📅", label: "Events" },
+      { emoji: "🙈", label: "Debts" },
+      { emoji: "💁‍♀️", label: "Customers" },
+    ],
+    general: [
+      { icon: LightbulbIcon, label: "Activity", notifications: "3" },
+      { icon: SquareCheckBigIcon, label: "Tasks", notifications: "4" },
+      { icon: UserIcon, label: "Contacts", notifications: "" },
+    ],
+  }
+
+  return (
+    <VStack
+      as="nav"
+      borderRight="1px solid"
+      borderRightColor="border"
+      h="full"
+      minH="700px"
+      w="300px"
+    >
+      <LinkBox
+        as="article"
+        cursor="pointer"
+        maxW="sm"
+        transition="all 0.1s"
+        _hover={{
+          background: ["#F5F5F5", "gray.800"],
+          textDecor: "none",
+        }}
+      >
+        <LinkOverlay href="#" onClick={(event) => event.preventDefault()}>
+          <HStack gap="md" p="md">
+            <Avatar
+              name="Hirotomo Yamada"
+              src="https://avatars.githubusercontent.com/u/84060430?v=4"
+            />
+            <HStack gap="lg">
+              <Box>
+                <Text as="b" fontSize="sm">
+                  Hirotomo Yamada
+                </Text>
+                <Text color="gray.400" fontSize="xs">
+                  hirotomo.yamada@avap.co.jp
+                </Text>
+              </Box>
+              <ChevronRightIcon />
```

Comment:
ChevronRightIcon is too far to the right.
<img width="302" alt="image" src="https://github.com/user-attachments/assets/5183d0fd-b02f-4d81-b0b9-9f26ccee68fd">
<img width="307" alt="image" src="https://github.com/user-attachments/assets/994d4ec8-f087-40c1-8ed3-d7485bdabf95">

Reply:
Thank you for pointing that out! I used `lg` for the gap and it shows properly on my end (picture below), but now I changed it to `md` to see if it works.

<img width="311" alt="Screenshot 2024-10-22 at 11 51 06 PM" src="https://github.com/user-attachments/assets/a69b8931-da26-4270-b8cc-9b0db554c9e2">